### PR TITLE
[develop] Move include/bb2diaspora to src/

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #3878 

Addons counterpart: https://github.com/friendica/friendica-addons/pull/538

### CODE REVIEW

**Comment by annando** [comment]
Looks good. But please don't forget about the addons.

**Comment by MrPetovan** [comment]
It's available in friendica-addons already.

**Comment by annando** [comment]
Yeah, **after** I said so :stuck_out_tongue:

File: src/Content/Text/BBCode.php [hunk 1]
```diff
@@ -1978,4 +1982,148 @@ private static function getAbstract($text, $addon = "")
...
+		$a = self::getApp();
+
+		$original_text = $text;
+
+		// Since Diaspora is creating a summary for links, this function removes them before posting
+		if ($for_diaspora) {
+			$text = self::removeShareInformation($text);
+		}
+
+		/**
```
Comment (rabuzarus) [inline src/Content/Text/BBCode.php]: please don't use `/**` for comments because `/**` would add the comment to the doxygen docu.

Comment (MrPetovan) [reply src/Content/Text/BBCode.php]: I didn't write it but I could have fixed it indeed.

**Pull request merged**